kubetest/anywhere.go

```diff
@@ -81,6 +83,7 @@ const kubernetesAnywhereConfigTemplate = `
 .phase2.kubelet_version="{{.KubeletVersion}}"
 .phase2.kubeadm.version="{{.KubeadmVersion}}"
 .phase2.kube_context_name="{{.KubeContext}}"
+.phase2.proxy_mode="{{.KubeproxyMode}}"
```

Comment:
unit-test?

Reply:
This args pass to kubernetes-anywhere, specify kube-proxy use ipvs mode.

Reply:
Thanks, I added a case for `proxy_mode=ipvs`.